ietf/meeting/migrations/0005_alter_session_agenda_note.py

```diff
@@ -0,0 +1,18 @@
+# Copyright The IETF Trust 2024, All Rights Reserved
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("meeting", "0004_session_chat_room"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="session",
+            name="agenda_note",
+            field=models.CharField(blank=True, max_length=512),
+        ),
+    ]
```

Comment:
Note: #7003 also has a 0005 migration. One of these will have to be updated to 0006, depending on merge order.